Dockerfile

```diff
@@ -1,15 +1,18 @@
 ARG BUILD_TYPE=dev
-ARG GOLANG=golang:1.19
-ARG BASE=fedora:36 
+ARG BUILDER_BASE=quay.io/confidential-containers/cloud-api-adaptor/golang-fedora:1.20.5-36
```

Comment:
Becomes `quay.io/confidential-containers/golang-fedora:1.20.5-36`

Reply:
I also just created https://quay.io/repository/confidential-containers/golang-fedora?namespace=confidential-containers

Reply:
I gave the robot write access to that repo!